battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/restart/RestartResponse.java

```diff
@@ -0,0 +1,7 @@
+package dreamteam.battleship.service.springcontroller.restart;
+
+/**
+ * Created by ehsan on 05.08.16.
+ */
+public class RestartResponse {
```

Comment:
Because the restart must retuns a type. so far we dont need any data from the restart service. thats why I created it but i left it empty. The service returns the type with empty content.